### DISCUSSION TITLE
Sub Folder for ROMs

### DISCUSSION THREAD

**Discussion by folhetinho** [eduardofilo/RG350_adam_image#162] [Ideas & Requests]
Hi guys,
Im in love with the CFW, it works perfect, but there's a thing thats a issue to me (and I think its for a few people as well)...
I like to Sort my roms (for each console) in sub folders, cause I play some hack roms and homebrews... so eg: SNES I have a subfolder for each region plus hacks / homebrews, but Im unable to do this on Adam Image :/ 

Is there any future plan on supporting sub folders?

**Comment by eduardofilo** [comment]
Hi.

We do not develop the pieces of software that are in the image (but Py Backup), we only integrate them.

SimpleMenu, which is the program that should support what you say, does recursively go through the subdirectories of the directories that are configured on each system. I suppose you mean that then it shows all the files mixed in a single list and it does not represent the subdirectories as you have them defined at the file system level, is that it?

**Reply by folhetinho** [reply]
Actually what I mean is that I would love to see my subfolders listed for each system...like:
When I open the SNES section, I'd like to see:
|SNES
         \American Roms 
                |Rom1
                |Rom2
         \Japanese Roms
                |Rom3
                |Rom4
         \Homebrews
         ...

But know you gave me the correct path to ask (staff of SimpleMenu, I think) ^^ so thanks for the info and happy gamming

**Reply by eduardofilo** [reply]
You can also take advantage of the different levels of organization of the systems in SimpleMenu in creative ways. That is, you can create new systems for each of those directories. Configured with the same emulator and changing something on the theme cover to distinguish them.
